### PR TITLE
[docs]: Updated the documentation for the Timer Component #10846

### PR DESCRIPTION
**Pull request by krishnadev7**
my previous pr was closed accidently and i am submiting the updated changes.
[link to the previous pr](https://github.com/ToolJet/ToolJet/pull/10863)

_Updated Review changes_

- removed internal.sh file from docker
- removed rest of the rows from styles table and updated description
- changed fx, count down and count up to bold from single tick
- updated time fromat `HH.MM.SS.MS` from `HH.mm.ss.SS`
- rechecked the events table and updated the description

### CODE REVIEW

File: docs/docs/widgets/timer.md [hunk 1]
```diff
@@ -2,90 +2,92 @@
...
+| <div style={{ width:"100px"}}> Property </div> | <div style={{ width:"100px"}}> Description </div>                                                                                                                                                        |
+| :--------------------------------------------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Default value                                  | It specifies the initial value of timer. Format is: `HH.MM.SS.MS`.                                                                                                                                       |
+| Timer type                                     | It specifies If its a upward or downward counter. Select **Count Up** or **Count Down** from the dropdown or you can click on **Fx** to programmatically define the values **countUp** or **countDown**. |
```
Comment (RudraPratik30) [inline docs/docs/widgets/timer.md:17]: Hey, the f in the fx should be small and not Fx, please check for this in the whole doc and both the version.

File: docs/docs/widgets/timer.md [hunk 2]
```diff
@@ -2,90 +2,92 @@
...
+| <div style={{ width:"100px"}}> Devices </div> | <div style={{ width:"100px"}}> Description </div> | <div style={{ width:"100px"}}> Expected Value </div>                                                                              |
+| :-------------------------------------------- | :------------------------------------------------ | :-------------------------------------------------------------------------------------------------------------------------------- |
+| Show on desktop                               | Makes the component visible in desktop view.      | You can set it with the toggle button or dynamically configure the value by clicking on **fx** and entering a logical expression. |
+| Show on mobile                                | Makes the component visible in mobile view.       | You can set it with the toggle button or dynamically configure the value by clicking on **fx** and entering a logical expression. |
 
 </div>
 
-<div style={{paddingTop:'24px', paddingBottom:'24px'}}>
+<div style={{paddingTop:'24px'}}>
```
Comment (RudraPratik30) [inline docs/docs/widgets/timer.md:80]: The div should start after the hr tag

**Review by RudraPratik30 — CHANGES_REQUESTED** [review]
Hey @krishnadev7 please make the mentioned changes, thanks!

File: docs/docs/widgets/timer.md [hunk 3]
```diff
@@ -2,90 +2,92 @@
...
+| Visibility                                     | Controls component visibility. Toggle or set dynamically.     | Enable/disable the toggle button or dynamically configure the value by clicking on **fx** and entering a logical expression. |
+| Disable                                        | Enables or disables the component. Toggle or set dynamically. | Enable/disable the toggle button or dynamically configure the value by clicking on **fx** and entering a logical expression. |
+| Box shadow                                     | Sets the box shadow properties of the component.              | Select the box shadow color and adjust the related properties or programmatically set it using **fx**.                       |
 
 :::info
-Any property having `Fx` button next to its field can be **programmatically configured**.
+Any property having **Fx** button next to its field can be **programmatically configured**.
 :::
-
-</div>
```
Comment (RudraPratik30) [inline docs/docs/widgets/timer.md:91]: Please close the div tag initialized for the style section.

**Review by RudraPratik30 — DISMISSED** [review]
Hey @krishnadev7!
Approving the PR from my end, and forwarding it to the team for final review.
Your PR will be merged by the end of this week.
Thanks for contributing!
Cc: @PriteshKiri

**Review by karan-rathod-316 — APPROVED** [review]
Looks good to me.

**Comment by PriteshKiri** [comment]
fixes #10846

**Pull request merged**